code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -490,7 +490,7 @@
 	var/mob/living/L = target
 	var/input_rune_key = input(user, "Choose a rune to teleport to.", "Rune to Teleport to") as null|anything in potential_runes //we know what key they picked
 	var/obj/effect/rune/teleport/actual_selected_rune = potential_runes[input_rune_key] //what rune does that key correspond to?
-	if(!src || QDELETED(src) || !user || user.l_hand != src && user.r_hand != src || user.incapacitated() || !actual_selected_rune)
+	if(QDELETED(src) || !user || user.l_hand != src && user.r_hand != src || user.incapacitated() || !actual_selected_rune)
```

Comment:
```suggestion
	if(QDELETED(src) || QDELETED(actual_selected_rune) || !user || user.l_hand != src && user.r_hand != src || user.incapacitated())
```
The rune might not yet be null but can already be qdeleted here. Can try and repro this by opening the TP window. Removing the dest rune and then selecting said rune.
Could do the same for user but the check after the `!user` should catch that already

Reply:
I can't see the problem problem here.
The rune being deleted before you selected it was fixed by adding the `QDELETED()` check, so it'll just make the rune fail now.